server/routes/dashboard.ts

```diff
@@ -255,6 +292,7 @@ export default function dashboard(service: Services): Router {
       incidentTypes,
       statusItems,
       typesLookup,
+      checkboxLabel,
```

Comment:
just a suggestion, but maybe this var name could be a little more specific? the label relates to status filters of different kinds.